frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1085,7 +1085,13 @@ function ReaderFooter:addToMainMenu(menu_items)
                 end,
             },
             {
-                text = _("Font"),
+                text_func = function()
+                    local font_weight = ""
+                    if self.settings.text_font_bold == true then
+                        font_weight = ", " .. _("bold")
+                    end
+                    return T(_("Font (%1%2)"), self.settings.text_font_size, font_weight)
```

Comment:
I didn't see this on your screenshot. (Also, parentheses or colon?)

Reply:
Colon is better.
It's not in the screenshots, as I have missed that one at start.